content/1_General/Lambda_Funcs.mdx

```diff
@@ -74,50 +70,45 @@ If we add the following from the link above in C++14:
 
 ```cpp
 namespace std {
+	template<class Fun>
+	class YCombinatorResult {
```

Comment:
can you revert this? I think this broke the compile (and it was fine before)

Reply:
oh sorry
what was the original name?

Reply:
just copy it over again from the original file

Reply:
I'd prefer if this was left unchanged

Reply:
why's that?

Reply:
because if it wasn't broken why change it?

Reply:
didn't change much, just adjusted the naming to fit naming conventions

Reply:
How does making it uppercase fit naming conventions? Wouldn't making it lowercase be consistent with the rest of the C++ standard library?

Reply:
classes & structs are usually named using PascalCase, no?